pkg/common/key.go

```diff
@@ -0,0 +1,110 @@
+package common
+
+import (
+	"fmt"
+	"strings"
+
+	guuid "github.com/google/uuid"
+)
+
+const (
+	KEY_DELIMETER = "/"
+	LOCKS         = "_locks"
+	COMMENTS      = "_comments"
+	INTERNAL_DB   = "_"
+)
+
+var prefix string
+
+type Key struct {
+	Prefix    string
+	DBName    string
+	TableName string
+	// the id represents uuid for the rows and id for the comments and locks
+	ID string
+}
+
+func SetPrefix(prf string) {
+	prefix = prf
+}
+
+func GetPrefix() string {
+	return prefix
+}
```

Comment:
I rather you don't use a global variable ... it just makes multi-threading more difficult later

Reply:
1. We do support multi-threading now. Multiple clients and even the same client can send simultaneous requests to the server. 
 2. This variable is set during server initialization, before opening any sockets.
 3. It is one of the standard server design patterns 
 4. I would prefer do not use global variables either, but it would significantly complicate the design, and implementation . Where is your washing to simplicity?